extensions/schema-compare/src/test/testUtils.ts

```diff
@@ -109,7 +109,7 @@ export async function setDacpacEndpointInfo(path: string): Promise<mssql.SchemaC
 	return endpointInfo;
 }
 
-export async function setDatabaseEndpointInfo(): Promise<mssql.SchemaCompareEndpointInfo> {
+export function setDatabaseEndpointInfo(): mssql.SchemaCompareEndpointInfo {
```

Comment:
There are a few lines in schemaCompare.test.ts that needs to be fixed (await removed) wherever these two functions are called (setDatabaseEndpointInfo, setDacpacEndpointInfo).

Reply:
removed those